web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1271,6 +1271,35 @@ export const formatters: Formatters = {
     desc: 'Login Rule Deleted',
     format: ({ user, name }) => `User [${user}] deleted a login rule [${name}]`,
   },
+  [eventCodes.SAML_IDP_AUTH_ATTEMPT]: {
+    type: 'saml.idp.auth',
+    desc: 'SAML IdP authentication',
+    format: ({ user, success }) => {
+      const desc =
+        success === false
+          ? 'failed to authenticate'
+          : 'successfully authenticated';
+      return `User [${user}] ${desc} with the SAML IdP`;
```

Comment:
Can there be more than one SAML IdP? If so, should we include the name of the IdP that failed?

Reply:
At the moment there's only one.